Sources/BraveShared/BraveStrings.swift

```diff
@@ -1092,6 +1092,7 @@ extension Strings {
   public static let shareWithMenuItem = NSLocalizedString("ShareWithMenuItem", tableName: "BraveShared", bundle: .module, value: "Share with...", comment: "Title for sharing url menu item")
   public static let openExternalAppURLTitle = NSLocalizedString("ExternalAppURLAlertTitle", tableName: "BraveShared", bundle: .module, value: "Allow link to switch apps?", comment: "Allow link to switch apps?")
   public static let openExternalAppURLMessage = NSLocalizedString("ExternalAppURLAlertMessage", tableName: "BraveShared", bundle: .module, value: "%@ will launch an external application", comment: "%@ will launch an external application")
+  public static let openExternalAppURLHost = NSLocalizedString("openExternalAppURLHost", tableName: "BraveShared", bundle: .module, value: "%@ says:", comment: "This will be used to indicate which website is calling the action. %s will replace the name of the website for instance: 'google.com says:'")
```

Comment:
Instead of `%@ says:` in the message body I think it's better to move the website name to the `openExternalAppURLTitle` variable, so it would say `Allow %@ to switch apps?`. What do you think @soner-yuksel ?

Reply:
Great idea. Fix at https://github.com/brave/brave-ios/pull/6704/commits/09b557867514bbac486f6986add5ab644c3c4b30